src/applications/personalization/rated-disabilities/sass/circular-progress.scss

```diff
@@ -0,0 +1,42 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
+.circular-progress {
+  background-color: transparent;
+  height: 185px;
+  margin: 0 auto;
+  position: relative;
+  width: 185px;
+}
+
+.circular-progress .circle {
+  height: 185px;
+  left: 0;
+  position: absolute;
+  top: 0;
+  width: 185px;
+}
+
+.disability-rating {
+  color: $color-gray-dark;
+  position: absolute;
+  text-align: center;
+  top: 62px;
+  width: 185px;
+}
+
+.fill {
+  fill: transparent;
+  stroke: $color-link-default;
+  stroke-dashoffset: 125;
+  stroke-width: 3;
+}
+
+.center {
+  fill: transparent;
+}
+
+.empty {
+  fill: transparent;
+  stroke: $color-gray-lighter;
+  stroke-width: 3;
+}
```

Comment:
This file can probably be removed since circular progress bar is no longer going to be used.

Reply:
Yep, removed in commit 0e211ef